pkg/controller/atlasmap/atlasmap_controller.go

```diff
@@ -147,5 +172,94 @@ func (r *ReconcileAtlasMap) Reconcile(request reconcile.Request) (reconcile.Resu
 		}
 	}
 
+	openShiftSemVer, err, result, done := r.getClusterVersionSemVer(err, reqLogger)
+	if done {
+		return result, err
+	}
+	constraint, _ := semver.NewConstraint(">= 4.3")
+	isOpenShift43Plus := constraint.Check(openShiftSemVer)
+
+
+	// Add link to OpenShift console
+	if isOpenShift43Plus {
+		route, reconcileResult, err, doneRoute := r.getRoute(err, request, reqLogger)
+		if doneRoute {
+			return reconcileResult, err
+		}
+
+		reconcileResult, err, done := r.CreateConsoleLink(request, err, reqLogger, route)
+		if done {
+			return reconcileResult, err
+		}
+	}
+
 	return reconcile.Result{}, nil
 }
+
+func (r *ReconcileAtlasMap) getClusterVersionSemVer(err error, reqLogger logr.Logger) (*semver.Version, error, reconcile.Result, bool) {
```

Comment:
How does this handle vanilla k8s installations (that aren't OCP)? Check out operator-utils /utils.go/IsOpenShift & LookupOpenShiftVersion and let's evaluate if that tooling is still relevant or can be caught up as there are still a lot of teams using the IsOpenShift check & some using it for versioning as well.

I'm honestly not sure if these features translate back to k8s, but if not, we should probably gate-check that.

In particular, let's look at the bug [mentioned here](https://github.com/RHsyseng/operator-utils/blob/master/internal/platform/platform_versioner.go#L110) and see if we can clean up the util code and maybe do away with the hard-coded version mapping altogether? Not sure we can, but thought you may want to get familiar with that code and see if it has a home here.

Reply:
@jeremyary: thanks, I'll check those utils methods and try to validate against k8s as well.

Reply:
I verified it won't work with K8s. I will replace with utils methods

Reply:
I reported a breaking change in the operator-sdk testing framework (https://github.com/operator-framework/operator-sdk/pull/2731) which is blocking me to use operator-util methods.